bin/ch-fromhost

```diff
@@ -145,7 +145,7 @@ enqueue_file () {
     old_ifs="$IFS"
     IFS="$newline"
     d="${dest:-$2}"
-    debug "queueing files"
+    debug "enqueue file(s)"
```

Comment:
I’d leave this as “queueing files” because “enqueue” is a rather esoteric word and I’m not sure we should give it to users.

Reply:
Aw, I like it. This is debug output, are you sure?

Reply:
Never mind; I'm convinced.